airflow/providers/amazon/aws/hooks/datasync.py

```diff
@@ -29,10 +29,13 @@ class AWSDataSyncHook(AwsBaseHook):
     """
     Interact with AWS DataSync.
 
+    Additional arguments (such as ``aws_conn_id``) may be specified and
+        are passed down to the underlying AwsBaseHook.
```

Comment:
Same here.

You need to indent in cases where you are using rst "functions" like `.. seealso` or lists, but not plain text.

Reply:
Thanks, fixed up

Reply:
Docs build is failing since I removed the indent:
https://travis-ci.org/apache/airflow/jobs/658184121?utm_medium=notification&utm_source=github_status
Adding it back, lets see